operator/v1/types_machineconfiguration.go

```diff
@@ -56,3 +66,94 @@ type MachineConfigurationList struct {
 	// Items contains the items
 	Items []MachineConfiguration `json:"items"`
 }
+
+type ManagedBootImages struct {
+	// machineManagers can be used to register machine management resources for boot image updates. The Machine Config Operator
+	// will watch for changes to this list. Only one entry is permitted per type of machine management resource.
+	//
+	// +optional
+	// +listType=map
+	// +listMapKey=resource
+	// +listMapKey=apiGroup
+	MachineManagers []MachineManager `json:"machineManagers"`
+}
+
+// MachineManager describes a target machine resource that is registered for boot image updates. It stores identifying information
+// such as the resource type and the API Group of the resource. It also provides granular control via the selection field.
+type MachineManager struct {
+	// resource is the machine management resource's type.
+	//
+	// Valid value(s): machinesets
+	//
+	// * machinesets means that the machine manager will only register resources of the type MachineSet, which may belong to MachineAPI or ClusterAPI.
```

Comment:
This won't render nicely in `kubectl explain`, make it more of a paragraph and more like prose. Lets not mention Cluster API just yet since it's not being supported just yet
```suggestion
	// resource is the machine management resource's type.
	// The only current valid value is machinesets.
	// machinesets means that the machine manager will only register resources of the kind MachineSet.
```